tests/test_qaes.py

```diff
@@ -9,7 +9,7 @@ def setUp(self) -> None:
         self.AES_KEY = '0' * 32
         self.cipher = AESCipher(self.AES_KEY)
 
-    def test_not_equal_encryption(self):
+    def test_not_equal_encryption_on_randomly_mode(self):
```

Comment:
Maybe adding a test with a small plain is a _nice to have_! I faced a failure with this plain: `"a"`

Reply:
Added.